src/AppBundle/Resources/config/repositories.yml

```diff
@@ -1,6 +1,8 @@
 services:
   app.repository.user:
     class: AppBundle\Model\User\UserRepository
+    tags:
+      - { name: app.tag.repository, entity: Account:User }
```

Comment:
well this seems to be something I didn't see while rebasing the old changes onto master, this ``app.tag.repository`` functionality has been removed and simplified already